roles/common/defaults/main.yml

```diff
@@ -257,7 +262,11 @@ gi_patches:
   - { category: "RU", base: "19.3.0.0.0", release: "19.7.0.0.200414", patchnum: "30783556", patchfile: "p30783556_190000_Linux-x86-64.zip", patch_subdir: "/30899722", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
   - { category: "RU", base: "19.3.0.0.0", release: "19.8.0.0.200714", patchnum: "31326369", patchfile: "p31326369_190000_Linux-x86-64.zip", patch_subdir: "/31305339", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
   - { category: "RU", base: "19.3.0.0.0", release: "19.9.0.0.201020", patchnum: "31720429", patchfile: "p31720429_190000_Linux-x86-64.zip", patch_subdir: "/31750108", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
+# Warning: GIRU 32126842 for 19.10.0.0.210119 has critical regressions and password protected. Development recommends 19.11.0.0.210420 Patch 32545008 instead of the JAN 2021 quarter RU
+  - { category: "RU", base: "19.3.0.0.0", release: "19.10.0.0.210119", patchnum: "32126842", patchfile: "p32126842_190000_Linux-x86-64.zip", patch_subdir: "/32226239", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
+#  - { category: "RU", base: "19.3.0.0.0", release: "19.11.0.0.210420", patchnum: "32578973", patchfile: "p32578973_190000_Linux-x86-64.zip", patch_subdir: "/32545008", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
 
+# data structure "rdbms_patches" carries the OJVM patches that go into RDBMS_HOME
```

Comment:
What does it mean to "carry" a patch?  Looking at the first patch 28729262, it looks like the regular PSU.

Reply:
That's right. The key: `patchnum: "29255947"` holds the patchnum for the PSU in earlier releases where `patch_subdir: ""` is empty. For the combo patches, the `patchnum` key holds the combo patch and the `patch_subdir` holds the OJVM RU.

Reply:
Sure, please see if the following is clear:
# data structure "rdbms_patches" contains the OJVM RU patch sub_dir that's derived from combo OJVM patchnum

Reply:
Removing as discussed